2022/week14/week14.tex

```diff
@@ -88,7 +90,10 @@ \section{Las Vegas}
 
 \begin{Exercice}[10 minutes]\textbf{Un point dans un cercle unitaire : Python} \optionnel\\
     Les algorithmes de Monte Carlo sont des algorithmes probabilistes dont la sortie peut être incorrecte avec une certaine probabilité, qui est généralement faible. En revanche, un algorithme de Las Vegas est un algorithme probabiliste qui trouve toujours le bon résultat lorsqu'il existe. Son inconvénient est que sa complexité temporelle ne peut être garantie à l'avance car elle dépend des données passées en paramètres.\\ \\
-    L'objectif de cet exercice est de programmer un algorithme probabiliste permettant de donner un point contenu dans un cercle unitaire. 
+    L'objectif de cet exercice est d'écrire un algorithme probabiliste permettant de générer les coordonnées d'un point contenu dans un cercle unitaire. Pour cela, vous pouvez générer des points \lstinline{n} fois jusqu'à ce que vous trouviez un point qui soit contenu dans le cercle.
```

Comment:
Ca serait peut-être intéressant pour cet exercise de demander à l'étudiant d'afficher le nombre d'iterations avant d'avoir trouvé le bon nombre. Comme ca on peut montrer que ce nombre d'itérations peut être énorme si on n'a pas de chance